### PR TITLE
add catalog

### PR DESCRIPTION
**Pull request by dimarogkov**
https://dimarogkov.github.io/layout_catalog
https://dimarogkov.github.io/layout_catalog/report/html_report

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,304 @@
...
+          <li class="header__item">
+            <a href="index.html" class="header__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+
+      <section class="cards">
```
Comment (polosanya) [inline src/index.html:54]:
```suggestion
    <main>
      <section class="cards">
```
Don't leave empty lines between parent and child elements

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    font-weight: 700;
+    line-height: 16px;
+    color: $color-white;
+    text-transform: uppercase;
+    padding: 11.2px 20px;
+    border-radius: 5px;
+    border: 1px solid $color-blue;
+    background-color: $color-blue;
+    will-change: transform;
+    transition: color 0.4s, background-color 0.4s, transform 0.4s;
```
Comment (polosanya) [inline src/styles/blocks/card.scss:102]: It would be great to make transition duration a variable too

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: $color-white;
+    text-transform: uppercase;
+    padding: 11.2px 20px;
```
Comment (polosanya) [inline src/styles/blocks/card.scss:97]: image

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  position: relative;
+  display: block;
+  padding: 50px 40px;
+
+  &__block {
+    position: relative;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-gap: 46px 50px;
+    width: 942px;
```
Comment (polosanya) [inline src/styles/blocks/card.scss:11]: What is this value? Create a variable for it or calculate it

File: src/styles/blocks/header.scss [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+
+  &__logo {
+    position: relative;
+    width: 40px;
+    min-width: 40px;
+    height: 40px;
+    margin: 10px 0;
+    transition: all 0.3s;
```
Comment (polosanya) [inline src/styles/blocks/header.scss:19]: Transition all is not good for performance. Better specify the exact properties.
Also, transition duration should be the same everywhere

File: src/styles/blocks/header.scss [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+    &:hover {
+      opacity: 0.7;
+    }
+
+    img {
```
Comment (polosanya) [inline src/styles/blocks/header.scss:25]: image

**Review by polosanya — CHANGES_REQUESTED** [review]
Good job! Let's improve your code a bit)

**Review by VitaliyBondarenko1982 — APPROVED** [review]

**Pull request closed without merging**